package.json

```diff
@@ -11,86 +11,85 @@
     "@fortawesome/free-regular-svg-icons": "^5.15.3",
     "@fortawesome/free-solid-svg-icons": "^5.15.3",
+    "@glimmer/component": "^1.0.0",
+    "@glimmer/tracking": "^1.0.0",
     "@turf/area": "^6.0.1",
     "@turf/line-distance": "^4.7.3",
-    "babel-eslint": "^8.0.0",
     "babel-plugin-transform-object-rest-spread": "^7.0.0-beta.3",
     "broccoli-asset-rev": "^3.0.0",
     "broccoli-clean-css": "^2.0.1",
     "ember-async-await-helper": "0.1.0",
-    "ember-auto-import": "^1.2.19",
+    "ember-auto-import": "^1.5.3",
     "ember-clean-project": "1.0.1",
-    "ember-cli": "~3.10.1",
+    "ember-cli": "~3.19.0",
     "ember-cli-app-version": "^3.2.0",
-    "ember-cli-babel": "^7.7.3",
+    "ember-cli-babel": "^7.20.5",
     "ember-cli-bundle-analyzer": "0.1.0",
     "ember-cli-code-coverage": "1.0.0-beta.7",
     "ember-cli-concat": "1.3.0",
-    "ember-cli-dependency-checker": "^3.1.0",
+    "ember-cli-dependency-checker": "^3.2.0",
     "ember-cli-dependency-lint": "1.1.3",
-    "ember-cli-eslint": "^5.1.0",
     "ember-cli-fastboot": "2.1.1",
-    "ember-cli-fastboot-testing": "0.0.17",
     "ember-cli-foundation-6-sass": "0.0.25",
-    "ember-cli-htmlbars": "^3.0.1",
-    "ember-cli-htmlbars-inline-precompile": "^2.1.0",
-    "ember-cli-inject-live-reload": "^1.8.2",
-    "ember-cli-mirage": "1.0.0",
+    "ember-cli-htmlbars": "^5.1.2",
+    "ember-cli-inject-live-reload": "^2.0.2",
+    "ember-cli-mirage": "^2.4.0",
     "ember-cli-netlify": "^0.1.0",
     "ember-cli-nouislider": "1.0.0",
     "ember-cli-preprocess-registry": "3.1.2",
     "ember-cli-sass": "^6.0.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-tree-shaker": "0.2.0",
-    "ember-cli-uglify": "^2.1.0",
+    "ember-cli-uglify": "^3.0.0",
     "ember-cli-update": "0.22.0",
     "ember-composability-tools": "0.0.11",
-    "ember-composable-helpers": "2.3.1",
+    "ember-composable-helpers": "3.2.0",
     "ember-concurrency": "1.0.0",
     "ember-concurrency-decorators": "1.0.0-beta.4",
     "ember-concurrency-retryable": "0.6.0",
     "ember-content-placeholders": "^1.0.0",
-    "ember-copy": "1.0.0",
-    "ember-data": "~3.10.0",
-    "ember-data-model-fragments": "4.0.0",
+    "ember-data": "~3.19.0",
+    "ember-data-model-fragments": "5.0.0-beta.8",
     "ember-decorators": "6.0.0",
     "ember-decorators-polyfill": "1.0.5",
     "ember-element-resize-detector": "0.2.1",
-    "ember-export-application-global": "^2.0.0",
-    "ember-fetch": "^5.1.1",
+    "ember-export-application-global": "^2.0.1",
+    "ember-fetch": "^5.1.3",
     "ember-g-recaptcha": "^1.3.0",
     "ember-in-viewport": "^3.1.3",
-    "ember-load-initializers": "^2.0.0",
-    "ember-local-storage": "1.7.0",
-    "ember-mapbox-composer": "0.2.3",
-    "ember-mapbox-gl": "0.14.0",
+    "ember-load-initializers": "^2.1.1",
+    "ember-local-storage": "2.0.2",
+    "ember-mapbox-composer": "git+https://github.com/allthesignals/ember-mapbox-composer#f7ef67053cb7e52459a2d3a42664723ce29846d5",
```

Comment:
You should be able to change this to `0.3.0` now. I published a [new version](https://www.npmjs.com/package/ember-mapbox-composer/v/0.3.0) on 11/3 that has the changes from your most recent PR in that repo. Let's pin this dependency to exactly that version.